CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- fix(firestore): use hostname from env variable (#184)
```

Comment:
Maybe make this more descriptive and around the user journey like "Correct the hostname for Firestore events in the emulator"